src/www/interfaces.php

```diff
@@ -2129,7 +2129,7 @@ function toggle_allcfg() {
                             </div>
                             <div class="hidden" data-for="help_for_dhcp_mode">
                               <?= gettext('The basic mode auto-configures DHCP using default values and optional user input.') ?><br/>
-                              <?= gettext('The advanced mode does not provide any default values, you will need to fill out any values you would like to use.') ?></br>
+                              <?= gettext('The advanced mode does not provide any default values, you will need to fill out any values you would like to use.') ?><br>
```

Comment:
now there's mixed style &lt;br&gt; vs &lt;br/&gt; within this file. I'm not sure which one is "better" *shrugs*